lib/challenge.js

```diff
@@ -135,14 +151,25 @@ function prompt7() {
  */
 
 //  define someNumber here
-var someNumber;
+var someNumber = Math.floor((Math.random() * 100)+ 1)
```

Comment:
Great job here, I would recommend use `let` instead of `var` as `var` is a bit outdated and `let` and `const` are more up to date